scripts/demo_content.py

```diff
@@ -0,0 +1,407 @@
+"""
+Drop any test database tables (user, item, order, orderitem)
+and supply a new one db with new down-to-earth data.
+"""
+
+from peewee import SqliteDatabase
+
+from models import User, Item, Order, OrderItem
+
+
+db = SqliteDatabase('database.db', autocommit=False)
+
+
+def main():
+
+    def drops_all_tables(database):
+        if db.database == 'database.db':
+            tables = db.get_tables()
+            for table in tables:
+                if table == 'item':
+                    Item.drop_table()
+                if table == 'order':
+                    Order.drop_table()
+                if table == 'user':
+                    User.drop_table()
+                if table == 'orderitem':
+                    OrderItem.drop_table()
+
+    def create_my_tables(database):
+        if db.database == 'database.db':
+            User.create_table(fail_silently=True)
+            Item.create_table(fail_silently=True)
+            Order.create_table(fail_silently=True)
+            OrderItem.create_table(fail_silently=True)
+
+    drops_all_tables(db)
+    create_my_tables(db)
+
+    User.create(
```

Comment:
Ma non c'era un modo per generare utenti in un bel for?
Cioe' la prima cosa che mi e' venuta in mente:
https://pypi.python.org/pypi/names/